### PR TITLE
Update Video Codec SDK

### PR DESCRIPTION
**Pull request by deiteris**
The main changes are:
1. Minimum driver requirement for Windows is 522.25 and for Linux 520.56.06 (https://docs.nvidia.com/video-technologies/video-codec-sdk/read-me/).
2. Existing combined presets are deprecated and split on quality and tuning presets. Corresponding preset option has been changed and low latency tuning is used.
3. NV_ENC_PARAMS_RC_CBR_LOWDELAY_HQ is deprecated and now has separate options (CBR + 2-pass encoding + I frame bits to P frame bits ratio).
4. Added more NvEnc options (tuning presets, weighted prediction, spatial and temporal adaptive quantization, multipass encoding configuration).

Current defaults are optimized for speed (P1 quality preset, low latency tuning preset, 2-pass encoding with quarter resolution).

GTX 1650, h264, 100% res, unlimited bitrate:
test

### CODE REVIEW

File: alvr/session/src/settings.rs [hunk 1]
```diff
@@ -22,9 +22,9 @@ pub enum FrameSize {
 #[derive(SettingsSchema, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase", tag = "type", content = "content")]
 pub enum NvencPreset {
-    LowLatencyDefault = 0,
-    LowLatencyHighQuality = 1,
-    LowLatencyHighPerformance = 2,
+    Quality = 0,
+    Balanced = 1,
+    Speed = 2,
```
Comment (zarik5) [inline alvr/session/src/settings.rs:27]: Can you merge this with the equivalent option for AMF?

**Review by zarik5 — CHANGES_REQUESTED** [review]

File: alvr/session/src/settings.rs (hunk 1 above)
Comment (zarik5) [reply alvr/session/src/settings.rs:27]: Can you also expose more encoding options? like 1-pass or 2-pass encoding (i'm not sure if that's a thing, i didn't read the new header, but just to give you an idea).

**Review by zarik5 — APPROVED** [review]

**Comment by zarik5** [comment]
Looks good. I don't have time to test this thoroughly but bugs can be fixed as they get reported.

**Comment by deiteris** [comment]
Then it's ready to merge. Didn't notice any issues on my end when I was testing all the settings.

**Pull request merged**